src/main/java/seedu/mark/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.mark.logic.commands;
+
+import static seedu.mark.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import seedu.mark.logic.commands.commandresult.CommandResult;
+import seedu.mark.logic.commands.exceptions.CommandException;
+import seedu.mark.model.Model;
+import seedu.mark.storage.Storage;
+
+/**
+ * Exports Mark's {@code Bookmark} data into a specific file.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Exports all of Mark's data to the specified file.\n"
+            + "Parameters: FILENAME\n"
+            + "Example: " + COMMAND_WORD + " savedBookmarks";
```

Comment:
Do we need to specify the file extension here?

Reply:
Generally is file extension included in the filePath?

Reply:
The file extension should be automatically appended to the user's choice of `FILENAME`. I'll correct the import and export command parsers.